### DISCUSSION TITLE
ArcLengthDimension with two arcs

### DISCUSSION THREAD

**Discussion by RealCommGithub** [haplokuon/netDxf#452] [Q&A]
I have a circle formed by two arcs:
-   First with StartAngle = 270, EndAngle = 90
-   Second with StartAngle = 90, EndAngle = 270

I used the ArcLengthDimension to draw the dimensions. I used the constructor with arc, offset and style.

What I espected to get was this: https://ibb.co/4YCRwcV
But my result is this: https://ibb.co/R3B4Pfg

The right arc doesn't have the DimensionLength. It seems like netDxf draw always in the same direction.

**Comment by haplokuon** [comment]
You will have to provide a full sample that shows your issue. I have not been able to reproduce it. This sample works just fine.
```c#
double radius = 4;
Vector2 center = new Vector2(1, 2);

Arc arc1 = new Arc(center, radius, 90, 270) { Color = AciColor.Blue };
Arc arc2 = new Arc(center, radius, 270, 90) { Color = AciColor.Red };
ArcLengthDimension dim1 = new ArcLengthDimension(arc1, radius + 1) { Color = AciColor.Cyan };
ArcLengthDimension dim2 = new ArcLengthDimension(arc2, radius + 1) { Color = AciColor.Magenta };

ArcLengthDimension dim3 = new ArcLengthDimension(center, radius, 90, 270, radius + 1) { Color = AciColor.Cyan };
ArcLengthDimension dim4 = new ArcLengthDimension(center, radius, 270, 90, radius + 1) { Color = AciColor.Magenta };

DxfDocument doc = new DxfDocument();
doc.Entities.Add(arc1);
doc.Entities.Add(arc2);
doc.Entities.Add(dim1);
doc.Entities.Add(dim2);
doc.Entities.Add(dim3);
doc.Entities.Add(dim4);
doc.Save("test.dxf");
```
